Snippets/Templates/Templates_1/Templates_1.csproj

```diff
@@ -1,7 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net462</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
The docker template is only going to be in the 2.0 version of the templates package, so these new snippets should be added to a new Templates_2 folder/project, which means these changes should be reverted.